src/components/App/App.js

```diff
@@ -13,90 +14,96 @@ class App extends Component {
       foundPokemon: [],
       error: null,
       // what will we need for favoriting?
-    }
+    };
   }
 
   componentDidMount() {
-    getPokedexData()
-    .then(data => {
+    getPokedexData().then((data) => {
       this.setState({
-        pokeDex: data.results
-      })
-    })
+        pokeDex: data.results,
+      });
+    });
   }
-
+  // i think we will modify this function ? or maybe not just add a function inside that calls the rigth api
   addPokemon = (queriedPokemon) => {
-    const foundPokemon = this.validatePokemonData(queriedPokemon)
+    const foundPokemon = this.validatePokemonData(queriedPokemon);
     this.setState({
-      foundPokemon: [foundPokemon]
-    })
+      foundPokemon: [foundPokemon],
+    });
     //set state with the validated pokemon
-  }
+  };
 
   validatePokemonData = (queriedPokemon) => {
-    const lowerCaseInput = queriedPokemon.queriedPokemon.toLowerCase()
+    const lowerCaseInput = queriedPokemon.queriedPokemon.toLowerCase();
 
     const verifiedName = this.state.pokeDex.find((pokemon, index) => {
-      let lowerCaseName = pokemon.name.toLowerCase()
-
-      if (lowerCaseName.includes(lowerCaseInput) && lowerCaseInput !== '' && lowerCaseInput !== undefined) {
-        return pokemon
-      } else if (parseInt(lowerCaseInput)){
-        if (index+1 === parseInt(lowerCaseInput)){
-          return pokemon
+      let lowerCaseName = pokemon.name.toLowerCase();
+
+      if (
+        lowerCaseName.includes(lowerCaseInput) &&
+        lowerCaseInput !== "" &&
+        lowerCaseInput !== undefined
+      ) {
+        return pokemon;
+      } else if (parseInt(lowerCaseInput)) {
+        if (index + 1 === parseInt(lowerCaseInput)) {
+          return pokemon;
         }
       }
     });
 
-    console.log(verifiedName)
+    console.log(verifiedName);
 
     if (verifiedName === undefined) {
-      return this.setState({ error: "Not  a valid name, try again" })
-      console.log('No Good Name!')
+      return this.setState({ error: "Not  a valid name, try again" });
```

Comment:
Maybe we change this to a binary (true or false) and we can say, if error is true, render some message or maybe even a searchError component?

Reply:
Good idea. I think that's a great place to go from the start we have.